### ISSUE TITLE
Libreoffice AppImage: Every Minor Update Creates New User profile

### ISSUE THREAD

**Issue by linuunix** [antoniofaccioli/libreoffice-appimage#18]
Description:
Under ~/.config/libreoffice/ I have several folders: 6.4.3 6.4.4 etc. There should actually be only one, namely 4. It seems every minor update create a new user directory named to the running LO version.

Steps to Reproduce:
1.Download LibreOffice (fresh) from https://www.libreoffice.org/download/appimage/
2.Make it executable and run it.
3.

Actual Results:
A new config directory named as LO version is created under ~/.config/libreoffice/

Expected Results:
https://wiki.documentfoundation.org/UserProfile#GNU.2FLinux states that there should be one and ONLY ONE directory, namely 4 which corresponds to LO versions 4 to 7.
Create the 4 directory if doesn't exist, other ways use the existing one.
Don't create directories named after Its version, other ways there would be thousands of dirs by time, keeping in mind that every such directory holds plenty of other sub directories. 

Reproducible: Always

User Profile Reset: Yes

Additional Info:
Version: 6.4.3.2
Build ID: 747b5d0ebf89f41c860ec2a39efd7cb15b54f2d8
CPU threads: 4; OS: Linux 5.6; UI render: default; VCL: kf5; 
Locale: de-DE (de_DE.UTF-8); UI-Language: en-US
Calc: threaded

**Comment by probonopd** [comment]
This is a feature, not a bug.

The LibreOffice team wanted it this way so that new/experimental versions can't accidentally mess up with existing settings on the system.

I think this line does it:

https://github.com/antoniofaccioli/libreoffice-appimage/blob/cec7a3fe03640df771a051a7c41ecfa53b9de92c/make_libreoffice_appimage#L367

**Comment by linuunix** [comment]
You are right, just tested it by commenting out that line, and see the standard behavior is back...Thanks for the hint.

> The LibreOffice team wanted it this way
Is this documented somewhere?

**Comment by smehrbrodt** [comment]
> The LibreOffice team wanted it this way so that new/experimental versions can't accidentally mess up with existing settings on the system.

You could as well use a different folder for the AppImage, but keep the version number.
This way, the AppImage would not interfere with other LibreOffice installations, but still users would have all their settings in place when updating.

For example, change the path from `$SYSUSERCONFIG/libreoffice` to `$SYSUSERCONFIG/libreoffice_appimage`
